modules/oadp-backup-restore-for-large-usage.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+// about-installing-oadp.adoc
+
+:_mod-docs-content-type: CONCEPT
++[id="oadp-backup-restore-for-large-usage_{context}"]
+= NodeAgent CPU for large usage
+
+Testing shows that increasing `NodeAgent` CPU can significantly improve backup and restore times when using {oadp-first}.
+
+It is not recommended to use Kopia without limits in production environments on nodes running production workloads due to Kopia’s aggressive consumption of resources.
+
+However, running Kopia with limits that are too low results in CPU limiting and slow-performing backups and restore situations.
+
+The testing undertaken observed that running Kopia with 20 cores and 32 Gi memory supportedbackup and restore operations of over 100GB of data, multiple namespaces, or over 2000 pods in a single namespace.
```

Comment:
NIT: missing space between `supported` and `backup`

Reply:
+1